qt/widgets/common/src/FitPropertyBrowser.cpp

```diff
@@ -3279,6 +3281,38 @@ void FitPropertyBrowser::fit() {
   doFit(maxIterations);
 }
 
+/**
+ * Function to toggle the visibility of the settings browser
+ */
+void FitPropertyBrowser::toggleSettingsBrowserVisible() {
+  if (m_browser->isItemVisible(m_settingsGroup)) {
+    m_browser->setItemVisible(m_settingsGroup, false);
+  } else {
+    m_browser->setItemVisible(m_settingsGroup, true);
+  }
+}
+
+/**
+ * Function to toggle the visibility of a settings group property
+ * exposed to python
+ */
+void FitPropertyBrowser::removePropertiesFromSettingsBrowser(
+    const QStringList &propsToRemove) {
+  // get settings properties
+  QList<QtProperty *> props = m_settingsGroup->property()->subProperties();
+  for (auto &prop : props) {
```

Comment:
I'm not 100% clear on the implementation details but could this use `QList::contains` to avoid the nested for loop?

Reply:
Thanks was looking for something like this that would let me search through the properties by name - but I can use this to remove the loop over the QStringList of properties to remove.